packages/wrangler/src/user/whoami.ts

```diff
@@ -69,13 +76,30 @@ function printTokenPermissions(user: UserInfo) {
 			`🔓 To see token permissions visit https://dash.cloudflare.com/profile/api-tokens.`
 		);
 	}
-	logger.log(
-		`🔓 Token Permissions: If scopes are missing, you may need to logout and re-login.`
-	);
+	logger.log(`🔓 Token Permissions:`);
```

Comment:
should the order of these two logs be reversed, now that we're listing the missing token scopes?